sunbird_mcp/src/server.py

```
@@ -0,0 +1,375 @@
+Key Features:
+- Search for educational content with various filters
+- Retrieve detailed metadata for specific content items
+- Supports pagination, sorting, and field selection
+- Validates input parameters for API requests
+"""
+
+import json
+import logging
+import aiohttp
+import asyncio
+from typing import Dict, Any, List, Optional, Union, Tuple
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)